contracts/extensions/DeltaNeutralBasisTradingStrategyExtension.sol

```diff
@@ -1446,12 +1446,44 @@ contract DeltaNeutralBasisTradingStrategyExtension is BaseExtension {
     /**
      * Validate an ExchangeSettings struct settings.
      */
-    function _validateExchangeSettings(ExchangeSettings memory _settings) internal pure {
+    function _validateExchangeSettings(ExchangeSettings memory _settings) internal view {
         require(_settings.twapMaxTradeSize != 0, "Max TWAP trade size must not be 0");
         require(
             _settings.twapMaxTradeSize <= _settings.incentivizedTwapMaxTradeSize,
             "Max TWAP trade size must not be greater than incentivized max TWAP trade size"
         );
+
+        bytes memory data;
+
+        // For a single hop trade, trade data bytes length is 44. 20 source/destination token address + 3 fees + 20 source/destination token
+        // address + 1 fixInput bool. For multi-hop trades, trade data bytes length is greater than 44.
+        // `buyExactSpotTradeData` is trade data for an exact output trade. And exact output paths are reversed in UniswapV3.
+        data = _settings.buyExactSpotTradeData;
+        require(
+            data.length >= 44
+            && data.toAddress(0) == strategy.spotAssetAddress
+            && data.toAddress(data.length - 21) == address(collateralToken),
+            // && data.toBool(data.length - 1) == false,                           // FixIn is false; since exactOutput
+            "Invalid buyExactSpotTradeData data"
+        );
+
+        // `sellExactSpotTradeData` is trade data for an exact input trade.
+        data = _settings.sellExactSpotTradeData;
+        require(
+            data.length >= 44
+            && data.toAddress(0) == strategy.spotAssetAddress
+            && data.toAddress(data.length - 21) == address(collateralToken),
+            // && data.toBool(data.length - 1) == true,                            // FixIn is false; since exactInput
+            "Invalid sellExactSpotTradeData data"
+        );
+
+        data = _settings.buySpotQuoteExactInputPath;
+        require(
+            data.length >= 43                                                       // No FixIn bool at end
```

Comment:
Because fixIn bool is required only for data param that is passed to the trade module (& then to adapter). While above be passed to UniV3 quoter, and is hence only the UniV3 path.